plugins/core/hu.bme.mit.gamma.action.model/src/hu/bme/mit/gamma/action/util/ActionModelValidator.java

```diff
@@ -192,4 +193,25 @@ public ProcedureDeclaration getContainingProcedure(Branch branch) {
 		} 
 		throw new IllegalArgumentException("Unknown container for Branch.");
 	}
+	
+//////////////////////////////////////////////////////////////////////
+	public Collection<ValidationResultMessage> checkIfStatement(IfStatement ifs){
+		Collection<ValidationResultMessage> validationResultMessages = new ArrayList<ValidationResultMessage>();
+		// all Branch
+		for(Branch branch : ifs.getConditionals()) {
```

Comment:
Here and everywhere else: Please, conform to the rule "DO add a single space after keywords like for, do, while, if and else." defined in https://github.com/ftsrg/gamma/blob/master/docs/Coding-conventions.md